README.md

```diff
@@ -212,6 +212,7 @@ the `env_variables` section of `app/app.yaml`.
 *   `STORAGE_ENGINE`: storage engine to use for storing metric metadata, 
     defaults to `datastore`.  
     `datastore` - use AppEngine Datastore
+    `boltdb` - use [BoltDB](https://github.com/etcd-io/bbolt) via [BoltHold](https://github.com/timshannon/bolthold)
```

Comment:
In rendered markdown this will get merged with the previous line, so perhaps you might want to separate it somehow.

Reply:
Ta! Fixed.